client/src/components/pages/HomePage.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+const HomePage = () => {
+  return (
+    <>
+      <main>
+        <h1>Welcome to the Phone Cave!</h1>
+        <h2>
+          Here you can see a list of our awesome <b>phones</b>!
+        </h2>
+      </main>
+      <nav>
+        <Link to="/phones">List of Phones</Link>
```

Comment:
suggesttion : As you are using a util to store your path in utils/path. i think it will be a good idea to use it here too, insetad of hardcoded to="/phones"

Reply:
I was using but I hardcoded it when I was testing the new react-router-dom. I haven't time to use it properly in the HomePage component. Time flied yesterday...